.github/workflows/ci.yml

```
@@ -12,15 +12,27 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        redis-version:
+          - 4.0
+          - 4.1
+          - 4.2
+          - 4.3
+          - 4.4
+          - 4.5
+          - 4.6
+          - latest
         ruby-version:
           - 2.3
           - 2.4
           - 2.5
           - 2.6
           - 2.7
-          - "3.0"
+          - 3.0
           - 3.1
           - head
```

The latest redis gems don't support ruby 2.3.  Need to add exclusions to the test matrix... or drop support for ruby 2.3. 😉

```suggestion
          - head
        exclude:
          - { ruby-version: 2.3, redis-version: 4.5 }
          - { ruby-version: 2.3, redis-version: 4.6 }
          - { ruby-version: 2.3, redis-version: latest }
```